src/pyotp/__init__.py

```diff
@@ -1,18 +1,16 @@
+import random
+
 from pyotp.otp import OTP
 from pyotp.hotp import HOTP
 from pyotp.totp import TOTP
-import utils
-
-import base64
-import random
-
+from . import utils
 
 VERSION = '1.3.0'
 
 
-def random_base32(length=16, random=random.SystemRandom(),
-                  chars=base64._b32alphabet.values()):
+def random_base32(length=16, r=random.SystemRandom(),

```

Comment:
The kwarg name should stay the same for API compatibility. Use `import random as _random` to avoid the name conflict.